extract.js

```diff
@@ -151,6 +151,17 @@ function loadBabelOpts(opts) {
 		}
 	}
 
+	// avoid conflicting with the legacy decorators plugin
+	if (opts.plugins && opts.plugins.findIndex((p) => p.key === 'proposal-decorators') > -1) {
+		const index = opts.parserOpts.plugins.findIndex(
+			(p) => Array.isArray(p) && p[0] === 'decorators',
+		);
+
+		if (index > -1) {
+			opts.parserOpts.plugins.splice(index, 1);
```

Comment:
This removes the following `decorator` plugin to make `loadBabelOpts()` respected user's `.babelrc`.
https://github.com/stylelint/postcss-css-in-js/blob/09af2eb5a1ba41279e099d095c268e170f02e88b/extract.js#L97